TesseractOCR/G8Tesseract.h

```diff
@@ -28,7 +28,8 @@
  *  Clear any library-level memory caches.
  *  There are a variety of expensive-to-load constant data structures (mostly
  *  language dictionaries) that are cached globally. This function allows the
- *  clearing of these caches.
+ *  clearing of these caches. It's safe to call this method, while a recognition is in progress.
```

Comment:
@ws233 can you make sure documentation/code doesn't extend beyond 80 characters? If the text would go beyond 80 characters, you should have it start on the next line (in the case of a comment, with another ` * `) and continue from there.

You can turn on a guideline for this using this method: http://grammarofdev.blogspot.com/2014/02/in-xcode-5-turn-on-80-column-line-guide.html